x/nodes/keeper/reward_test.go

```diff
@@ -422,3 +422,89 @@ func TestKeeper_rewardFromRelaysPIP22EXP(t *testing.T) {
 		})
 	}
 }
+
+func TestKeeper_RewardForRelaysPerChain(t *testing.T) {
+	Height_PIP22 := int64(3)
+	Height_RTTM2 := int64(10)
```

Comment:
DItto: What's the `2` for?

Reply:
I naively added 2 to distinguish per-chain RTTM from the original RTTM.  It's cryptic.  Let's change it to `PerChainRTTM`.

Reply:
Definitely adds a lot more sense!